examples/launch_height_baselines.py

```diff
@@ -88,7 +81,7 @@
         trial_backend = LocalBackend(entry_point=str(entry_point))
 
         stop_criterion = StoppingCriterion(
-            max_wallclock_time=5, min_metric_value={metric: -6.0}
+            max_wallclock_time=20, min_metric_value={METRIC_ATTR: -6.0}
```

Comment:
For posterity: increasing `max_wallclock_time` from `5` to `20` means there is more time for any errors to be thrown. `5` seconds was too short and the example was "succeeding" when in fact, with a slightly longer runtime it would fail, before the fix in this PR (#561)